projects/ontimize-web-ngx/src/lib/components/o-service-base-component.class.ts

```diff
@@ -183,6 +185,11 @@ export class OServiceBaseComponent implements ILocalStorageComponent, OnChanges
     } catch (e) {
       // no parent form
     }
+    try {
+      this.expandableContainer = this.injector.get(OExpandableContainerComponent);
+    } catch (e) {
+      // No parent OExpandableContainerComponent
+    }
```

Comment:
Why not reuse previous try-catch

Reply:
because if there is an exception inside the try-catch the code is not executed